File: BowlPicks.Api/Models/CreateAccountModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;

namespace BowlPicks.Api.Models
{
    public class CreateAccountModel
    {
        public string Username { get; set; }
        public string Password { get; set; }
        public string ConfirmPassword { get; set; }
        public string Email { get; set; }
    }
}

File: BowlPicks/app/models/CreateAccountModel.js
"use strict";

if (typeof BowlPicks == "undefined") {
    window.BowlPicks = {};
}

if (typeof BowlPicks.Models == "undefined") {
    BowlPicks.Models = {};
}

if (typeof BowlPicks.Models.CreateAccountModel == "undefined") {
    BowlPicks.Models.CreateAccountModel = function () {
        this.Username = null;
        this.Password = null;
        this.ConfirmPassword = null;
        this.Email = null;
    };
}